.github/workflows/ci.yml

```
@@ -44,19 +44,17 @@ jobs:
           - 'amazonlinux-2'
           - 'centos-7'
           - 'centos-8'
-          - 'debian-8'
           - 'debian-9'
-          - 'fedora-31'
+          - 'debian-10'
+          - 'fedora-32'
```

This way you don't need to change it on new releases.
```suggestion
          - 'fedora-latest'
```